users/konstantin/rules.mk

```diff
@@ -0,0 +1,16 @@
+BOOTMAGIC_ENABLE   = no
```

Comment:
no? Not even lite?

Reply:
I really dislike the Bootmagic user experience, Command is a great alternative for me 😃

Reply:
Also with lite I'm kind of scared of accidentally putting the board into bootloader mode when holding it and plugging it in.